Okta.AspNetCore.Mvc.IntegrationTest/Controllers/HomeController.cs

```diff
@@ -1,38 +0,0 @@
-﻿// <copyright file="HomeController.cs" company="Okta, Inc">
```

Comment:
I pinged Laura offline to ask why we were removing the controllers and static files from `Okta.AspNetCore.Mvc.IntegrationTest`. It sounds like these aren't actually needed by the integration tests since the tests are only concerned with startup logic.

Just a nitpick but I'd recommend adding a line to the PR description to explain the motivation for removing those files. That seems like a change which isn't directly related to the .NET core version upgrade so its good to note why it is happening.